code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -262,15 +262,7 @@
 	slot_flags = ITEM_SLOT_BACK
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/dual
 	sawn_desc = "Omar's coming!"
-	obj_flags = UNIQUE_RENAME
```

Comment:
This is for changing the name/description with a pen, unrelated to the visual reskin.